src/frontend/web_application/src/components/Button/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import Icon from '../Icon';
+import { Icon } from '../';
```

Comment:
this is a thing we didn't talk.

I found that sometimes the import doesn't work using a directory index.

for example if three components are in the same directory like Menu, Button & Icon, the import will work for Menu using Icon because Icon has already  been parsed by webpack and it won't for Button using Icon.

It may change with versions of webpack, babel etc. 

Well I think to make sure this works is to not using import from index when we are in the directory.

**OK:**

```
// file: src/scene/Foo/presenter.jsx
import { Button } from '../components';

// file: src/components/Button/index.jsx
import Icon from '../Icon';
```

**NOT OK**
```
// file: src/scene/Foo/presenter.jsx
// It works correctly but not efficient
import Button from '../components/Button';

// file: src/components/Button/index.jsx
// It might be completely broken
import { Icon } from '../';
```

Reply:
ok

Reply:
and what about sub-directories of `/components/`?
Ex. for file: `src/components/Foo/components/Bla/presenter.jsx`